compiler-rt/lib/asan/scripts/asan_symbolize.py

```diff
@@ -779,9 +788,13 @@ def __str__(self):
             arch=self.arch,
             start_addr=self.start_addr,
             end_addr=self.end_addr,
-            module_path=self.module_path
-            if self.module_path == self.module_path_for_symbolization
-            else "{} ({})".format(self.module_path_for_symbolization, self.module_path),
+            module_path=(
+                self.module_path
+                if self.module_path == self.module_path_for_symbolization
+                else "{} ({})".format(
+                    self.module_path_for_symbolization, self.module_path
+                )
+            ),
```

Comment:
just checking - there's no change here, just formatting, right?
Was this intended or applied by some kind of auto-formatter? Just wondering if we should minimize the diff by reverting this.

As an aside - this assignment to `module_path` is moderately complex. I wonder if we could make it simpler by expanding the ternary expression into a proper `if-else` block. But that's probably out of scope.

Reply:
This was the result of running `black` on the file, since I'm touching the file. I can revert if minimal diff is preferred.

Reply:
I can't say authoritatively one way or another, but I'd imagine a minimal diff will make it easier to approve.